900.version-fixes/s.yaml

```diff
@@ -236,6 +236,7 @@
 - { name: stp,                         verlonger: 3,                 ruleset: freebsd,     incorrect: true }
 - { name: stp,                         verpat: "[0-9]{4,}",                                incorrect: true }
 - { name: stp,                         verpat: "201[56].*",                                outdated: true }
+- { name: strace,                      verpat: ".*ab.*", devel: true } # the version is technically never defined and rather incorrect than devel
```

Comment:
Better mark as incorrect + additional untrusted rule for sisyphus and fedora

Reply:
See 800b04a